general/src/filter/common.rs

```diff
@@ -98,7 +98,8 @@ impl<'de> Deserialize<'de> for GlobPatterns {
 ///
 /// Ported from Sentry's same-named "enum". The enum variants are fed into outcomes in kebap-case
 /// (e.g.  "browser-extensions")
-#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Serialize, Hash)]
+#[serde(rename_all = "kebab-case")]
```

Comment:
With the discussed changes, we can likely revert all changes in this file. It's no longer required to serialize this directly.

If serialization is still required in the end, please implement it using `.as_str()` rather than deriving it a second time.

Reply:
ok, done